conda/environment-dev.yaml

```diff
@@ -20,3 +20,9 @@ dependencies:
   - r-data.table
   - r-testthat >= 3.0.0
   - r-readr
+  - r-pastecs
+  - r-rlist
+  - r-rann

```

Comment:
I'm not a conda expert but I had to include r-pastecs, r-rlist and r-rann in meta.yaml to be able to build the package

Reply:
Yeah, in order to build the conda package, these would have to be included in meta.yaml - thanks for catching that one!

Reply:
See [4c2c301](https://github.com/RECETOX/recetox-xMSannotator/pull/17/commits/4c2c301bcd45887f4f63fab828776515b6fe921d)